src/cli/cli.cpp

```diff
@@ -5364,6 +5364,39 @@ template <> otError Interpreter::Process<Cmd("neighbor")>(Arg aArgs[])
             OutputLine("| %5lu |", ToUlong(neighborInfo.mAge));
         }
     }
+#if OPENTHREAD_CONFIG_UPTIME_ENABLE
+    /**
+     * @cli neighbor uptime
+     * @code
+     * neighbor uptime
+     * | RLOC16 | Extended MAC     | Age            | Uptime         |
+     * +--------+------------------+----------------+----------------+
+     * | 0xbc02 | f62e76dea80fb595 |             70 |            186 |
+     * | 0xbc03 | a245d666cd4b0bfa |             59 |             59 |
+     * | 0xc800 | 66183d4298be21da |             20 |            635 |
+     * @endcode
+     * @par
+     * Print the uptime and age of neighbors. Info per neighbor:
+     * - RLOC16
+     * - Extended MAC address
+     * - Age (seconds since last heard from neighbor)
+     * - Uptime (seconds since link establishment with neighbor)
+     */
+    else if (aArgs[0] == "uptime")
+    {
+        static const char *const kUptimeTableTitles[]       = {"RLOC16", "Extended MAC", "Age", "Uptime"};
+        static const uint8_t     kUptimeTableColumnWidths[] = {8, 18, 16, 16};
```

Comment:
16 chars seems larger than needed for a 32-bit value.

16 chars seems smaller than needed for a 64-bit value.

Reply:
Will change in next push.
I may actually change the table to use a nicer format (`"<hh>:<mm>::<ss>"`) for durations (to make it easier to read).

Reply:
`<hh>:<mm>::<ss>` would be nicer!

Reply:
Added this format in the new pushed commit.